src/parser/jobs/dnc/modules/Technicalities.tsx

```diff
@@ -0,0 +1,262 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import _ from 'lodash'
+import React, {Fragment} from 'react'
+import {Icon} from 'semantic-ui-react'
+
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {RotationTable} from 'components/ui/RotationTable'
+import {getDataBy} from 'data'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {BuffEvent, CastEvent} from 'fflogs'
+import Module, {dependency} from 'parser/core/Module'
+import Combatants from 'parser/core/modules/Combatants'
+import {AoeEvent} from 'parser/core/modules/Combos'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import Timeline from 'parser/core/modules/Timeline'
+
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+import FeatherGauge from './FeatherGauge'
+
+// Harsher than the default since you'll only have 4-5 total windows anyways
```

Comment:
Can skip straight to medium if you think it's warranted

Reply:
Still wanted to leave the possibility of a herp or fat-finger or something and not be too mean about it. They get one slip-up though, after that it's Med+